### PR TITLE
Added tutorial for indexing to CSV files, locally

### PR DESCRIPTION
**Pull request by RaekwonIII**

### CODE REVIEW

File: docs/tutorials/index-to-local-csv-files.md [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+Next, let's created a new file, named `tables.ts`, in case the project should be expanded to multiple files. This is where it's possible to provide a filename for the CSV files, as well as configure their data structure, the same way as if they were a database table (the class name is no coincidence):
+
+```typescript
+import {Table, Column, Types} from '@subsquid/file-store-csv'
+
+export const Transfers = new Table(
+    'transfers.csv',
+    {
+        blockNumber: Column(Types.Integer()),
+        timestamp: Column(Types.DateTime()),
```
Comment (belopash) [inline docs/tutorials/index-to-local-csv-files.md:89]: Let's rename this type to `Timestamp`

File: docs/tutorials/index-to-local-csv-files.md [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+        timestamp: new Date(block.header.timestamp),
+        contractAddress: item.address,
+        from: from.toLowerCase(),
+        to: to.toLowerCase(),
+        amount: BigDecimal(value.toBigInt(), decimals).toNumber(),
+      });
+    }
+  }
+
+  ctx.store.Transfers.writeMany(transfers);
```
Comment (belopash) [inline docs/tutorials/index-to-local-csv-files.md:189]: no need to use writeMany, it's the same as
for () {
  ctx.store.Transfers.write(transfer)
}

**Comment by belopash** [comment]
bad idea to keep your python script in data folder

File: docs/tutorials/index-to-local-csv-files.md (hunk 2 above)
Comment (RaekwonIII) [reply docs/tutorials/index-to-local-csv-files.md:189]: Thanks for the tip, but it's still one line vs a for loop, I prefer it 🤷

File: docs/tutorials/index-to-local-csv-files.md (hunk 1 above)
Comment (RaekwonIII) [reply docs/tutorials/index-to-local-csv-files.md:89]: What do you mean? I can rename the field...the type is defined in the library...

File: docs/tutorials/index-to-local-csv-files.md (hunk 2 above)
Comment (belopash) [reply docs/tutorials/index-to-local-csv-files.md:189]: you already have for loop

Comment (belopash) [reply docs/tutorials/index-to-local-csv-files.md:189]: replace .push with .write

File: docs/tutorials/index-to-local-csv-files.md (hunk 1 above)
Comment (belopash) [reply docs/tutorials/index-to-local-csv-files.md:89]: Package updated

Comment (RaekwonIII) [reply docs/tutorials/index-to-local-csv-files.md:89]: Is it available on npm? Cause I don't see it
https://www.npmjs.com/package/@subsquid/file-store-csv

**Review by abernatskiy — APPROVED** [review]

**Pull request merged**